backend/Dockerfile.model_server

```diff
@@ -21,10 +21,12 @@ RUN apt-get remove -y --allow-remove-essential perl-base && \
 RUN python -c "from transformers import AutoModel, AutoTokenizer, TFDistilBertForSequenceClassification; \
 from huggingface_hub import snapshot_download; \
 AutoTokenizer.from_pretrained('danswer/intent-model'); \
-AutoTokenizer.from_pretrained('intfloat/e5-base-v2'); \
+AutoTokenizer.from_pretrained('nomic-ai/nomic-embed-text-v1'); \
+AutoTokenizer.from_pretrained('nomic-ai/nomic-bert-2048'); \
```

Comment:
to make this work while airgapped, you need to .from_pretrained and snapshot_download not only nomic-ai/nomic-embed-text-v1, but also nomic-ai/nomic-bert-2048.

It was hard to find the exact reasoning for this, but I'm pretty sure it has something to do with nomic-embed-text-v1 being built on top of nomic-bert-2048 and that it needs to run .py scripts located only in the the nomic-bert-2048 repo [here](https://huggingface.co/nomic-ai/nomic-bert-2048/tree/main)